easybuild/easyblocks/n/nvhpc.py

```diff
@@ -90,8 +91,8 @@ def __init__(self, *args, **kwargs):
         """Easyblock constructor, define custom class variables specific to NVHPC."""
         super(EB_NVHPC, self).__init__(*args, **kwargs)
 
-        # Potential improvement: get "Linux_x86_64" from easybuild.tools.systemtools' get_cpu_architecture()
-        self.nvhpc_install_subdir = os.path.join('Linux_x86_64', self.version)
+        architecture = 'Linux_%s' % platform.uname()[4]
```

Comment:
Can't we do what the comment suggests instead?

```suggestion
        architecture = 'Linux_%s' % easybuild.tools.systemtools.get_cpu_architecture()
```
+ change of import statement

Reply:
Should have commented instead of running with it. I tested with `get_cpu_architecture()` and on our `ppc64le` system it simply returned `POWER`. Since I needed something to reflect [`arch`](https://easybuild.readthedocs.io/en/latest/version-specific/easyconfig_templates.html) I simply copied its definition.

I'll add a comment to this effect so that it is clearer why I did this.